Unity/Assets/YarnSpinner/TheSecretLab.YarnSpinner.asmdef

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "TheSecretLab.YarnSpinner",
```

Comment:
Change name to "YarnSpinner"

Reply:
Hey, here is a weird one. If I do that, compilation of the YarnSpinner.asmdef silently fails (there is no compile error being logged into Unity's console). The only hint is that YarnSpinner Editor compilation fails due to YarnSpinner not being available.

If you drop the YarnSpinnerEditor asmdef, the you'll get clearable (meaning you can clear these compile errors which you usually cannot) compile errors like this:
```
Error: Could not load signature of NodeVisitedTracker:<Start>b__2_0 due to: Could not resolve type with token 0100000a (from typeref, class/assembly Yarn.Value, YarnSpinner, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null) assembly:YarnSpinner, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null type:Yarn.Value member:(null) signature:<none>
```
```
Unloading broken assembly Library/ScriptAssemblies/YarnSpinner.dll, this assembly can cause crashes in the runtime
```
```
Yarn.Unity.InMemoryVariableStorage' is missing the class attribute 'ExtensionOfNativeClass'!
```

I don't fully understand what is going on but my guess about what's causing this is that there's this "YarnSpinner/Scripts/DLLs/YarnSpinner.dll" file which now has the same name as the dll that will be created from the `YarnSpinner.asmdef`.  
So to avoid that name collision, I can think of two ways handling this:
* rename YarnSpinner.dll to Yarn.dll and Yarn's Unity related files are called YarnSpinner -> `Yarn` is the library name, `YarnSpinner` is the Unity integration
* leave YarnSpinner.dll and rename Yarn's Unity related files to YarnSpinnerUnity -> `YarnSpinner` is the library name, `YarnSpinnerUnity` is the Unity integration

Reply:
Huh, weird. I’ll take a look at this and see what the best solution is.

Reply:
What do you think about the second suggestion? Seems to be the easiest since you don't need to change the name of your existing library.

Reply:
I made the requested change but changed the name to `YarnSpinnerUnity` to avoid the name collision: ac9b44da31cfc95d01bc33ea284ef9135e09933b